crates/cli/tests/cli_run.rs

```diff
@@ -149,13 +150,7 @@ mod cli_run {
         // for some reason, llvm prints out this warning when targeting windows
         let ignorable = "warning: ignoring debug info with an invalid version (0) in app\r\n";
         let stderr = stderr.replacen(ignorable, "", 1);
```

Comment:
Can we also include all of this in the shared function?

Reply:
I kept it out since the `ignorable` substring differs by call site. Would you prefer to pass `ignorable` as another arg to `has_error`?

Reply:
I think they all should always be ignorable. I think they are just not in sync because they only get added as they are hit.

Reply:
Makes sense. Pushed an update. Note that this now logs the full stderr in the panic rather than the stripped version. Let me know if you'd prefer it like before.